Reqnroll.VisualStudio/ProjectSystem/Settings/ReqnrollProjectSettingsProvider.cs

```diff
@@ -210,13 +210,27 @@ private NuGetPackageReference GetSpecFlowPackage(IProjectScope projectScope,
 
     private string GetReqnrollConfigFilePath(IProjectScope projectScope)
     {
-        var projectFolder = projectScope.ProjectFolder;
         var fileSystem = projectScope.IdeScope.FileSystem;
-        return fileSystem.GetFilePathIfExists(Path.Combine(projectFolder,
-                   ProjectScopeDeveroomConfigurationProvider.ReqnrollJsonConfigFileName)) ??
-               fileSystem.GetFilePathIfExists(Path.Combine(projectFolder,
-                   ProjectScopeDeveroomConfigurationProvider.SpecFlowJsonConfigFileName)) ??
-               fileSystem.GetFilePathIfExists(Path.Combine(projectFolder,
-                   ProjectScopeDeveroomConfigurationProvider.SpecFlowAppConfigFileName));
+        var assemblyDirectory = Path.GetDirectoryName(projectScope.OutputAssemblyPath);
+
+        var configFilePath = GetConfigFileInPath(fileSystem, projectScope.ProjectFolder);
+
+        if (assemblyDirectory != null)
+        {
+          configFilePath ??= GetConfigFileInPath(fileSystem, assemblyDirectory);
+          configFilePath ??= GetConfigFileInPath(fileSystem, Path.Combine(assemblyDirectory, "..", ".."));
+        }
+        
+        return configFilePath;
+    }
+
+    private static string GetConfigFileInPath(IFileSystemForVs fileSystem, string projectFolder)
```

Comment:
I would rename the param `projectFolder` to just `folder`

Reply:
Fixed